packages/runtime-common/indexer.ts

```diff
@@ -412,28 +423,24 @@ export class Indexer {
   // client may be serving a live index or a WIP index that is being built up
   // which could have conflicting loaders. It is up to the caller to provide the
   // loader that we should be using.
-  async search(
+  private async _search(
     realmURL: URL,
     { filter, sort, page }: Query,
     loader: Loader,
-    opts?: QueryOptions,
-    // TODO this should be returning a CardCollectionDocument--handle that in
-    // subsequent PR where we start storing card documents in "pristine_doc"
-  ): Promise<{ cards: CardResource[]; meta: QueryResultsMeta }> {
+    opts: QueryOptions,
+    selectStatement: string,
```

Comment:
this would be better as a `CardExpression` type--then you can include parameters in the select statement if need be.